scripts/prepdocs.py

```diff
@@ -515,6 +516,27 @@ def read_files(
                 read_files(filename + "/*", use_vectors, vectors_batch_support)
                 continue
             try:
+                # if filename ends in .md5 skip
+                if filename.endswith(".md5"):
+                    print("Skipping md5 hash index.")
+                    continue
+
+                # if there is a file called .md5 in this directory, see if its updated
+                stored_hash = None
+                with open(filename, "rb") as file:
+                    existing_hash = hashlib.md5(file.read()).hexdigest()
```

Comment:
This should be called something like : new_hash. Here we are calculating a hash of a new file.